index.js

```diff
@@ -1,5 +1,5 @@
-import React, { Component } from 'react';
```

Comment:
Can you revise this commit so that the only changes are the ones related to clearing the interval and not formatting?